docs/tutorials/broadcast.md

```diff
@@ -8,148 +8,62 @@ import TabItem from "@theme/TabItem";
+const message = "Hello from XMTP!"; // Your broadcast message
+sendBroadcastMessage(recipients, message);
 ```
 
-</TabItem>
-</Tabs>
-
-## Best practices for broadcast messages
-
-- **Depending on where you’re based**, you could be subject to data privacy laws, including but not limited to GDPR and CCPA.
-
-- **If your app sends broadcast messages to your users**, get user consent before sending them broadcast messages. For example, you can request this consent during onboarding. Here's some example text you can build upon:
-
-  > (&nbsp;&nbsp;) **Yes**, I want to receive broadcast messages from &lt;app name&gt;.
-  >
-  > These messages may include updates, promotions, and other relevant information.
-  >
-  > You can unsubscribe at any time by adjusting your notification settings within the app or by replying “STOP” to a broadcast message.
-  >
-  > We value your privacy and will use your contact information only to send these broadcast messages.
-  >
-  > By clicking **Continue**, you confirm that you agree to receive broadcast messages from &lt;app name&gt;.
-  >
-  > (&nbsp;&nbsp;) **No**, I don’t want to receive broadcast messages from &lt;app name&gt;.
-  >
-  > [Cancel]&nbsp;&nbsp;&nbsp;[**Continue**]
-
-- **If your app allows your customers to send broadcast messages to their users**, be sure to advise your customers to get their users' consent before sending them broadcast messages. For example, you can provide this guidance during onboarding. Here's some example text you can build upon:
-
-  > By signing up to use this app, you agree to get consent from your users before sending them broadcast messages using this platform. When requesting user consent, let users know what kinds of message content they can expect to receive and how you'll use their contact information.
-  >
-  > Also be sure to provide users with a way to unsubscribe from your broadcast messages, such as via notification settings or by replying "STOP" to a broadcast message.
-
-- **If your app stores a signature on a server to read and send XMTP messages on behalf of your customer**, such as automated broadcast messages, be sure to let your customer know. Ideally, your app should also periodically auto-delete signatures and provide a way for a customer to manually revoke their signature. For example, you can provide this guidance as a part of the XMTP connection flow. Here's some example text and UX you can build upon:
+### Managing high volume broadcasts
 
-  For example:
+When planning to send broadcast messages at a high volume, it's crucial to consider XMTP's rate limits to ensure efficient and responsible use of the network. High volume broadcasts require careful strategy to avoid rate limiting issues and to maintain network health.
 
-  - Before connection:
+- **Adherence to Rate Limits**: Understand and respect XMTP's rate limits to prevent network overload and ensure your messages are delivered smoothly.
```

Comment:
```suggestion
- **Adherence to Rate Limits**: Understand and respect XMTP's [rate limits](/docs/faq#rate-limiting) to prevent network overload and ensure your messages are delivered smoothly.
```